### ISSUE TITLE
Mobile menu uses prohibited aria attributes

### ISSUE THREAD

**Issue by corneliusroemer** [loculus-project/loculus#2729]
From Google Pagespeed audit: https://pagespeed.web.dev/analysis/https-preview-update-loculus-4df831-pathoplexus-org/lqbf3u6gja?form_factor=mobile

<img width="1212" alt="Brave Browser 2024-09-09 11 37 54" src="https://github.com/user-attachments/assets/06644716-95ad-44f9-9dd2-f8e811fec8f7">

> Elements use prohibited ARIA attributes
> 
> Using ARIA attributes in roles where they are prohibited can mean that important information is not communicated to users of assistive technologies. Learn more about prohibited ARIA roles.
> Failing elements: main menu `<div class="relative" aria-label="main menu">`
